registry.md

```diff
@@ -56,16 +56,16 @@ And the possible parameters they each may have:
 
 | Parameter | Value Type                                    | Description                              |
 |:----------|-----------------------------------------------|------------------------------------------|
-| `network` | see [Ethereum Networks](#ethereum-networks) | The particular blockchain netowrk to use |
+| `network` | see [Ethereum Networks](#ethereum-networks) | The particular blockchain network to use |
 
 
 #### Ethereum Networks
 
-| Value     | Description                      | Network Id |
-|:----------|:---------------------------------|------------|
-| `regtest` | Private Ethereum regtest network | N/A        |
-| `ropsten` | Ropsten testnet                  | 3          |
-| `mainnet` | Ethereum mainnet                 | 1          |
+| Value     | Description                          | Network Id |
+|:----------|:-------------------------------------|------------|
+| `regtest` | Private Ethereum development network | N/A        |
```

Comment:
This could give the wrong impression that a regtest network does not have a network id. 
For our parity regtest (dev-chain) the network ID is 17.

Reply:
I plan to discuss this in tracker issue.